docs/maintainerflow.rst

```diff
@@ -0,0 +1,85 @@
+.. highlight:: none
+
+
+************************
+Workflow for maintainers
+************************
+
+In order to ease the process of maintaining, the following strategy is applied
+for the development of DFTB+ and related projects.
+
+Roles
+=====
+
+We have three groups of developers with different access rights to the
+repositories:
+
+- Coordinating developers, repository administrators: They are responsible for
+  strategic decisions. Some of them also represent the project in certain
+  commercial collaborations.
+
+- Core developers, repository maintainers: Experienced, skilled developers
+  contributing regularly to the projects and having a good overview over the
+  project. They are often responsible for certain set of features within the
+  code.
+
+- Contributors: casual contributors.
+
+
+Access Rights
+=============
+
+The groups as listed above have following rights:
+
+- Repository administrators: They have all possible rights on the repositories,
+  including the possibility to grant rights to other developers.
+
+- Repository maintainers: They have write access to project repositories. They
+  can review and merge pull requests (PR). They can write into the corresponding
+  repository of an of contributors during the review process. They have access
+  to the test server, so that they can execute extended tests on the code in a
+  pull request.
```

Comment:
Would go well together with enabling branch protection for the default/master branch.

Reply:
:heavy_check_mark: